BigQuery/src/LoadJobConfiguration.php

```diff
@@ -637,4 +637,46 @@ public function hivePartitioningOptions(array $hivePartitioningOptions)
 
         return $this;
     }
+
+    /**
+     * Sets whether to create a new session with random id.
+     * The created session id is returned as part of the SessionInfo
+     * within the query statistics.
+     *
+     * Example:
+     * ```
+     * $loadJobConfig->createSession(true);
+     * ```
+     *
+     * @param bool $createSession
+     * @return LoadJobConfiguration
+     */
+    public function createSession(bool $createSession)
```

Comment:
Do you think adding a default value of `true` makes sense?

Reply:
It may not be what users wants. This session is only useful (at least currently) when the customer wants to have temporary tables for loading. Every other case, the customer don't need this session at all. So, I think its better if we don't give any defaults.